.github/workflows/lock.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: "0 */12 * * *"
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ github.token }}
+          issue-lock-inactive-days: '30'
+          pr-lock-inactive-days: '30'
+          issue-lock-reason: 'resolved'
+          pr-lock-reason: 'resolved'
+
+          issue-lock-comment: 'This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.'
+          pr-lock-comment: 'This pull request has been automatically locked since there has not been any recent activity after it was closed. Please open a new pull request for related changes.'
```

Comment:
Better wording:

`This pull request has been automatically locked since there has not been any activity in it in the last 30 days. If you want to tell us about needed or wanted changes or if problem related to this code are discovered, feel free to open an issue or a new pull request.`